packages/graphql/lib/graphql.module.ts

```diff
@@ -157,6 +158,7 @@ export class GraphQLModule<
       ...options,
       typeDefs: mergedTypeDefs,
     });
+    GraphQLModule.logger.log(ROUTE_MAPPED_MESSAGE(options.path, RequestMethod.POST));
```

Comment:
What if someone registers a custom driver that doesn't require passing the `path` options property? We should add `if` condition her to make sure we don't log `undefined` values.

Reply:
Thanks for the review!
Please check back as I have corrected the issues you pointed out.